sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,263 @@
+# Writing Performance Tests
+
+## [Index](#index)
+
+- [Setting up the project](#Setting-up-the-project)
+  - [Track 2](#Setting-up-the-project)
+  - [Track 1](#For-perf-testing-track-1-version-of-the-same-package)
+- [Writing perf tests](#writing-perf-tests)
+  - [Entry Point](#Entry-point)
+  - [Base Class](#Base-Class)
+  - [Test File](#Test-file)
+  - [Custom Options](#custom-options)
+- [Executing the perf tests](#executing-the-perf-tests)
+  - [Command to run](#Command-to-run)
+  - [Adding Readme/Instructions](#Adding-Readme/Instructions)
+  - [Testing an older track 2 version](#Testing-an-older-track-2-version)
+
+## [Setting up the project](#Setting-up-the-project)
+
+To add perf tests for the `sdk/<service>/<service-sdk>` package, follow the steps below.
+
+1.  Create a new folder for the perf tests.
+
+    Path- `sdk/<service>/perf-tests/<service-sdk>`
+
+    (Create the `perf-tests` folder if that doesn't exist)
+
+2.  This new perf test project will be managed by the rush infrastructure in the repository, with the package name `@azure-tests/<service-sdk>`. To allow rush to manage the project, add the following entry in `rush.json`
+
+    ```
+        {
+          "packageName": "@azure-tests/perf-<service-sdk>",
+          "projectFolder": "sdk/<service>/perf-tests/<service-sdk>",
+          "versionPolicyName": "test"
+        }
+
+    ```
+
+3.  Tests will live under `sdk/<service>/perf-tests/<service-sdk>/test`
```

Comment:
I think we're supposed to start directing people to go to the `template` project. Perhaps we could setup a skeleton structure in there?

Reply:
Great point.. https://github.com/Azure/azure-sdk-for-js/issues/14117

Added a good-first-issue label.